_data/glossary/statefulset.yml

```diff
@@ -0,0 +1,18 @@
+id: statefulset
+name: StatefulSet
+formerly:
+- PetSet
+related:
+- Deployment
+- Pod
+tags:
+- Storage
+- Workload
+- API Object
+short-description: |
+  Manages the deployment and scaling of a set of replica Pods, *and provides guarantees about ordering*. It does so by maintaining a *unique*, sticky identity for each of its Pods.
```

Comment:
It's a bit awkward because I can see this being singular in some places but plural in others. Need to figure out a clean way for this.